### PR TITLE
Portfolio start

### PR DESCRIPTION
**Pull request by MarcoDDM**
For this feature branch I:

-add linters configuration.
-add styles for the mobile version.
-set up the HTML for the headline and the navbar.

### CODE REVIEW

**Review by iambenkis — CHANGES_REQUESTED** [review]
Hi @MarcoDDM , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that needs to satisfy the following requirements:
- [ ] Your ReadMe does not conform to the **new** [standardized ReadMe template](https://github.com/microverseinc/readme-template). Kindly ensure that all the headers from the [README template](https://github.com/microverseinc/readme-template) are  present in your repo, and then request another review

**[New ReadMe Template](https://github.com/microverseinc/readme-template):**
image

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">    <title>Marco's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo"><img src="images/logo.svg"></a>
+        <a href="#" class="hamburger"><img src="images/hamburger.svg"/></a>
```
Comment (tufoinnkuo10) [inline index.html]: - [x] great job.  for semantics reason the Hamburger icon should have a `nav `tag

File: index.html [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+    </header>
+
+    <div class="headline">
+        <h1>Marco Almada</h1>
+    </div>
+    <div class="description">
+        <h2>Hello! I am a software developer!
```
Comment (tufoinnkuo10) [inline index.html]: - [x] kindly put this in a `p `tag
- [x] please respect the figma font weight, size and line height

File: index.html [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Playfair+Display&display=swap" rel="stylesheet">    <title>Marco's Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com"><link rel="preconnect" href="https://fonts.gstatic.com" crossorigin><link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="header">
+        <a href="#" class="logo"><img src="images/logo.svg"></a>
+        <a href="#" class="hamburger"><img src="images/hamburger.svg"/></a>
+    </header>
+
```
Comment (tufoinnkuo10) [inline index.html:18]: - [ ] please ensure The container of the section is a `section` tag.

* It represents a generic standalone section of the document, so use it.

File: index.html [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <h1>Marco Almada</h1>
+    </div>
+    <div class="description">
+        <h2>Hello! I am a software developer! 
+            I can help you build a product, feature or website. 
+            Take a look of my works.
+            If you like what you see and have a project you need coded, 
+            don’t hesitate and contact me.
+        </h2>
+        <button class="collab">Start collaboration</button>
```
Comment (tufoinnkuo10) [inline index.html]: - [ ] kindly add a link placeholder to the button

File: index.html [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+    </div>
+
```
Comment (tufoinnkuo10) [inline index.html:31]: - [ ] great job. Please reduce the gap between the text and start collaboration button like that of Figma.
- [ ] also, you are missing the image under the start collaboration button . please, kindly add that.

yours
Screenshot (634)_LI
figma
Screenshot (633)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @MarcoDDM 

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally


Required Changes ♻️
Check the comments under the review
tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by algerina — APPROVED** [review]
# STATUS: Approved  🥳   🎉🎉🎉

### Hello @MarcoDDM 
 ✋🏼 Nice done with working on last review. I am impressed. Your project is great. There is nothing else to say other than... it's time to merge it!

### Cheers and Happy coding! 💻 👏



HIGHLIGHTS 
- No linter errors
- Descriptive PR and commits
- Your project looks Great!
___
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form.](https://airtable.com/shrQAqnBwek5a0O0s)_
___
_If anything seems to be unclear to you, reach out and ask anything by commenting on this Pull Request and mention me as [Algerina](https://github.com/algerina), I will receive a notification and will respond._

[GitHub](https://github.com/algerina) | [LinkedIn](https://linkedin.com/in/amel-khiri) | [Twitter](https://twitter.com/AmelQahwadji)

_Reviewed by [Amel Khiri ](https://github.com/algerina)_

**Pull request merged**